services/openshift/scripts/openshift_provision

```diff
@@ -9,6 +9,19 @@ export ORIGIN_DIR="/var/lib/origin"
 export OPENSHIFT_DIR=${ORIGIN_DIR}/openshift.local.config/master
 export KUBECONFIG=${OPENSHIFT_DIR}/admin.kubeconfig
 
+# Wait for container to show up
+ATTEMPT=0
```

Comment:
Should this not really be in the openshift script? Once provision is called I would expect that OpenShift is up and running